quickcheck-dynamic/src/Test/QuickCheck/StateModel.hs

```diff
@@ -210,26 +242,50 @@ lookUpVar env v = case lookUpVarMaybe env v of
 
 data WithUsedVars a = WithUsedVars VarContext a
 
+data Polarity
+  = PosPolarity
```

Comment:
suggestion:
```
data Polarity = Positive | Negative
```

Reply:
I think I had this at some point, but `Positive` clashed with QuickCheck's `Positive` 😅 Though that import could be hidden as the polarity shouldn't be used as a client of this library and shouldn't be exported in the primary interface.